app/views/sections/info/DispatchPlaceView.scala.html

```diff
@@ -35,7 +35,11 @@
     @formHelper(action = action, Symbol("autoComplete") -> "off") {
 
         @if(form.errors.nonEmpty) {
-            @govukErrorSummary(ErrorSummaryViewModel(form,  errorLinkOverrides = Map("value" -> "value_0")))
+            @defining({
+                DispatchPlace.options.headOption.flatMap(_.id).getOrElse("value_0")
```

Comment:
Even though we had a hardcoded `value_0` before I don't think it's worth keeping that as the only values it can be are `value_GB` or `value_XI`.